From a9af70031e426a152999c930f11c6998220456b3 Mon Sep 17 00:00:00 2001
From: Glenn Burkhardt <glenn.burkhardt@utas.utc.com>
Date: Tue, 5 Feb 2019 10:53:11 -0500
Subject: [PATCH] Use correct attribute name 'skipped' to be compatible with
 xUnit 2.3.1; see Jenkins issue 51914
 (https://issues.jenkins-ci.org/browse/JENKINS-51914) and nose issue 1083
 (https://github.com/nose-devs/nose/issues/1083); xUnit's officially supported
 JUnit schema are Ant junit and Maven Surefire

---
 nose/plugins/xunit.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nose/plugins/xunit.py b/nose/plugins/xunit.py
index 77e68e35..2ee72089 100644
--- a/nose/plugins/xunit.py
+++ b/nose/plugins/xunit.py
@@ -237,7 +237,7 @@ def report(self, stream):
             u'<?xml version="1.0" encoding="%(encoding)s"?>'
             u'<testsuite name="%(testsuite_name)s" tests="%(total)d" '
             u'errors="%(errors)d" failures="%(failures)d" '
-            u'skip="%(skipped)d">' % self.stats)
+            u'skipped="%(skipped)d">' % self.stats)
         self.error_report_file.write(u''.join([force_unicode(e, self.encoding)
                                                for e in self.errorlist]))
         self.error_report_file.write(u'</testsuite>')
